packages/core/src/request.ts

```diff
@@ -34,7 +35,9 @@ export function eventToSentryRequest(event: Event, api: API): SentryRequest {
   if (useEnvelope) {
     const envelopeHeaders = JSON.stringify({
       event_id: event.event_id,
-      sent_at: new Date().toISOString(),
+      // We need to add * 1000 since we devide it by 1000 by default but JS works with ms precision
```

Comment:
```suggestion
      // We need to add * 1000 since we divide it by 1000 by default but JS works with ms precision
```

Still don't understand why we multiply by 1000 though, doesn't `timestampWithMs` return a timestamp in ms, and you instantiate a Date object with a number in ms?

Reply:
JS + Sentry is bad combo ^^

timestamps by default in JS are in ms. `1589805085388`
Sentry can't work with that, only in seconds with ms precision `1589805085.388`

Maybe not the best function name but that's the reason.